gitpod-io/gitpod
# [local-preview] Warn and Confirm from user before proceeding

## Description


Currently, For Users with ARM CPU's, `local-preview` does not really work and its hard for the script to know as it runs inside docker (which could be a x86 VM).

This Updates the script to warn the users on the requirements before starting and running `local-preview`.

<img width="1382" alt="Screenshot 2022-09-20 at 6 27 14 PM" src="https://user-images.githubusercontent.com/7892868/191264639-588c612e-d229-4208-8848-861f514db8d9.png">

The default value is also `YES` which means even when the user presses enter, we proceed forward.

Signed-off-by: Tarun Pothulapati <tarun@gitpod.io>

## Related Issue(s)

Fixes #11792 

## How to test


Run

```
docker run -p 443:443 --privileged --name gitpod -it  --mount type=volume,source=gitpod,destination=/var/gitpod eu.gcr.io/gitpod-core-dev/build/local-preview:tar-lp-warn-confirm.28
```

## Release Notes

```release-note
[local-preview] Warn and Confirm from user before proceeding
```

## Documentation


## Werft options:

      Valid options are `all`, `workspace`, `webapp`, `ide`
